### DISCUSSION TITLE
kitty backgroud transparent

### DISCUSSION THREAD

**Discussion by NEX-S** [kovidgoyal/kitty#5778] [Q&A]
if we set the kitty's backgroud color the same as the vim/neovim's background color, we will get the transparent

eg.
```
neovim
hi Normal guibg=#232323 guifg=#ffffff

kitty
background #232323
```
we will get neovim transparent:
![image](https://user-images.githubusercontent.com/97107165/207638281-81c57926-b9f6-44c8-bb20-e68f8a54fd0c.png)

is this behavior a bug? if is, I think this "bug" is very useful cuz we can do some cool stuff depending on this behavior:
for example, we can set neovim's tabline transparent like this

![image](https://user-images.githubusercontent.com/97107165/207638657-f0b8f54f-c11c-4b08-a2e7-f0e3c5917bb5.png)

which is pretty coooooooooooool so I don't think this "bug" should get fixed :)

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.background_opacity

**Reply by NEX-S** [reply]
thx!
